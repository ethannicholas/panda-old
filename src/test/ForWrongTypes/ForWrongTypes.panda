for i:String, v:Int in 1 .. 10 {
}
for i:Int8, v:String in 1 .. 10 {
}
for v:Object in 1 .. 10 {
}