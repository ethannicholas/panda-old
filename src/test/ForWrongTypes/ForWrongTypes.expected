error: ForWrongTypes.panda:1:5: loop index must be an integer
error: ForWrongTypes.panda:3:13: elements of panda.collections.Array<panda.core.Int32> cannot be converted to panda.core.String
2 errors
